Frameworks/UIKit/UIWebView.mm

```diff
@@ -243,11 +243,31 @@ - (void)setDetectsPhoneNumbers:(BOOL)detect {
 }
 
 /**
- @Status Interoperable
+ @Status Gap
 */
 - (NSString*)stringByEvaluatingJavaScriptFromString:(NSString*)string {
-    NSString* ret = [_xamlWebControl invokeScript:@"eval" arguments:[NSArray arrayWithObject:string]];
-    return ret;
+    UNIMPLEMENTED_WITH_MSG(
+        "stringByEvaluatingJavaScriptFromString is not supported on our platform. Call evaluateJavaScript:completionHandler: instead.");
+    return StubReturn();
```

Comment:
This is hitting a build break;

D:\Build\Agent1\_work\2\s\Frameworks\UIKit\UIWebView.mm(251,12): error : use of undeclared identifier 'StubReturn' [D:\Build\Agent1\_work\2\s\build\UIKit\lib\UIKitLib.vcxproj]

Can you please add #include "StubReturn.h" just below the #include "Starboard.h"?